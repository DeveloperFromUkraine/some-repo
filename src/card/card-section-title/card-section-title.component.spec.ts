import { ComponentFixture, TestBed } from "@angular/core/testing";
import { CardSectionTitleComponent } from "./index";
import { ComponentTest } from "../../../test/test-bed/component";


describe('CardSectionTitleComponent', () => {
    let fixture: ComponentFixture<CardSectionTitleComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [CardSectionTitleComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(CardSectionTitleComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});