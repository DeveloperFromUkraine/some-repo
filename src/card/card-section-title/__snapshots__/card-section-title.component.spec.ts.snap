// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardSectionTitleComponent should match snapshot 1`] = `
<ign-card-section-title>
  <h2
    class="ign-card-section-title mat-card-subtitle"
  >
    
  </h2>
</ign-card-section-title>
`;
